### PR TITLE
Allow templates to use maxItems and minItems for Arrays

### PR DESCRIPTION
**Pull request by JasonNorth**
Allows maxItems and minItems to be used in templates.
This is dependent on a release of swagger-core version 1.5.11.
This is related to Issue 3925 but only addresses maxItems and minItems.

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/swagger-api/swagger-codegen/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/swagger-api/swagger-codegen/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

File: pom.xml [hunk 1]
```diff
@@ -853,7 +853,7 @@
         <swagger-parser-version>1.0.24-SNAPSHOT</swagger-parser-version>
         <scala-version>2.11.1</scala-version>
         <felix-version>2.3.4</felix-version>
-        <swagger-core-version>1.5.10</swagger-core-version>
+        <swagger-core-version>1.5.11</swagger-core-version>
```
Comment (JasonNorth) [inline pom.xml]: this needs a release of swagger-core 1.5.11

**Comment by JasonNorth** [comment]
I had a go at fixing the linking to my github account but I guess I may have made things worse...

**Comment by wing328** [comment]
@JasonNorth looks like the rebase didn't work as expected. I would suggest you close this PR and submit a new PR instead.

**Pull request closed without merging**